### ISSUE TITLE
Feature Case for exunit

### ISSUE THREAD

**Issue by keathley** [elixir-wallaby/wallaby#142]
We should include a `feature` exunit test type. Having a custom type will allow us to capture more information and use that information to create better screenshot names and capture screenshots on assertion failures.

**Comment by mhanberg** [comment]
@keathley,

Last night I took a stab at implementing this, but I think I may need some guidance.

Some problems I ran into:

1) Calling Ecto and Phoenix functions in the test case
2) How to know the name of the user's project (e.g., translate MyApp.Repo into <Actual app anme>.Repo>

I think I understand what information would be captureable to to enhance screenshots, but I'm curious what other info you had in mind.

Regarding 1), we could forgo the phoenix and ecto bits, which would at minimum stop the user from always needing to create an exunit case themself, but would still require it if they wanted the ecto sandboxing stuff.

Thanks
